e2e/testcases/otel_collector_test.go

```diff
@@ -49,7 +49,7 @@ const (
 	DefaultMonitorKSA             = "default"
 	MonitorGSA                    = "e2e-test-metric-writer"
 	MetricExportErrorCaption      = "One or more TimeSeries could not be written"
```

Comment:
Might need to change this line to lower case too

Reply:
That one still uses capitals in the example log i posted.

Reply:
Ack